_pages/WG6/Leuven/admin.md

```diff
@@ -10,7 +10,7 @@ breadcrumbs: true
 
 - [x] The chair asks the organizer the exact address, the starting and ending dates and times of the meeting, and the daily allowance. The daily allowance should cover the cost of an hotel night + the meals + local transport if hotels are far from the meeting place. It has to be fixed at the right level so that people get reimbursed of what they need to spend but not too much so that we can invite more people. Note that the daily allowance cannot be changed once invitations have been sent.
 
-- [ ] The organizer gives to the chair the exact address, and the starting and ending dates and times:
+- [x] The organizer gives to the chair the exact address, and the starting and ending dates and times:
```

Comment:
Please tick this box only when all the sub-items are ticked. The times are still missing but I guess it will be more or less something like 9:00-18:00, right?

Reply:
We will almost certainly stay within that interval, yes.